libs/application/templates/driving-license/src/fields/Congratulations.tsx

```diff
@@ -24,23 +20,39 @@ function Congratulations({
   field,
   application,
 }: PropTypes): JSX.Element {
-  const { formatMessage } = useLocale()
-
   return (
-    <Box paddingTop={4}>
-      <BulletList type="ul">
-        <Bullet>
-          {formatText(m.congratulationsBullet1, application, formatMessage)}
-        </Bullet>
-        <Bullet>
-          {formatText(m.congratulationsBullet2, application, formatMessage)}
-        </Bullet>
-        <Bullet>
-          {formatText(m.congratulationsBullet3, application, formatMessage)}
-        </Bullet>
-      </BulletList>
-      <Box marginTop={8}>
-        <img src="/assets/images/movingTruck.svg" alt="Skrautmynd" />
+    <Box paddingTop={2}>
+      <Box marginTop={2}>
+        <ContentBlock>
+          <AlertMessage
+            type="success"
+            title="Til hamingju"
+            message={`
+              fullnaðarskírteinið þitt hefur verið samþykkt og mun vera tilbúið
+              hjá Sýslumanni í Reykjavík eftir 4 daga. Þegar þú sækir nýtt skírteini
+              þarftu að skila inn bráðabirgðaskírteininu þínu.
+            `}
+          />
+        </ContentBlock>
+
+        <Box marginTop={4}>
+          <Accordion singleExpand>
+            <AccordionItem
+              id="id_1"
+              label="Hvað gerist næst?"
+              labelVariant="h3"
+            >
+              <Text>
+                Næsta sem gerist er að við ákveðum hvað á að standa í þessu
+                boxi.
+              </Text>
+            </AccordionItem>
+          </Accordion>
+        </Box>
+
+        <Box marginTop={4}>
+          <img src="/assets/images/movingTruck.svg" alt="Skrautmynd" />
```

Comment:
Consider removing text from `alt` as the image is only decorative also adding `role="presentation"` would make this more clear to e-readers